tgui/packages/tgui/interfaces/Stack.js

```diff
@@ -83,14 +75,14 @@ const Multipliers = (props, context) => {
     maxMultiplier,
   } = props;
 
-  let maxM = Math.min(maxMultiplier,
+  const maxM = Math.min(maxMultiplier,
     Math.floor(recipe.max_res_amount / recipe.res_amount));
 
-  let multipliers = [5, 10, 25];
+  const multipliers = [5, 10, 25];
 
   let finalResult = [];
 
-  for (let multiplier of multipliers) {
+  for (const multiplier of multipliers) {
```

Comment:
I am surprised const works in that context. Apparently it's not a bug.

Reply:
Tested various multipliers thoroughly with this and found no issue whatsover. Would it be better to revert it back to let, just in case? This exact context is also prosent in 2 more interfaces from what I have found, and they also have no issues with that.

Reply:
No, just commented because I always assumed for..of loops reassigned the entry, hence it had to use `let` in a similar way how traditional loops need it, but I was wrong for all these years.